packages/vite/src/node/__tests__/env.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { posix } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { describe, expect, test } from 'vitest'
+import { loadEnv } from '../env'
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url))
+
+describe('loadEnv', () => {
+  test('basic', async () => {
+    expect(loadEnv('development', posix.join(__dirname, './env')))
```

Comment:
I think we should use `path.join` instead of `path.posix.join`. `__dirname` is like `C:\foo\bar\` and `posix.join(__dirname, './env')` will be `C:\foo\bar\/env`. I does work it will be strange. (By using `path.join`, the result will be `C:\foo\bar\env`)

Reply:
I forgot about this, thanks for tell me.